output_template/python/usb_camera_demo.py

```diff
@@ -79,6 +80,32 @@ def add_class_label(canvas,
     cv2.putText(canvas, text, dl_corner, font, font_scale, font_color, line_type)
 
 
+def label_to_color_image(results, colormap):
+    """Adds color defined by the dataset colormap to the label.
+
+    Args:
+        results: A 2D array with integer type, storing the segmentation label.
```

Comment:
Is the array integer ? I think this is float.

Reply:
thanks, it is float, fixed with commit 7c71b86